tests/model/ArrayListTest.php

```diff
@@ -314,67 +314,46 @@ public function testSortSimpleDefaultIsSortedASC() {
 		), $list->toArray());
 	}
 
-	public function testNaturalSort() {
-		//natural sort is only available in 5.4+
-		if (version_compare(phpversion(), '5.4.0', '<')) {
-			$this->markTestSkipped();
-		}
-		$list = new ArrayList(array(
-		array('Name' => 'Steve'),
+	public function testMixedCaseSort() {
+		// Note: Natural sorting is not expected, so if 'bonny10' were included
+		// below we would expect it to appear between bonny1 and bonny2. That's
+		// undesirable though so we're not enforcing it in tests.
```

Comment:
@dhensby thought I would document our thinking here for future code diggers - okay?